lib/fastlane/plugin/firebase/actions/firebase_add_apple_store_id.rb

```diff
@@ -0,0 +1,104 @@
+module Fastlane
+  module Actions
+    class FirebaseAddAppleStoreIdAction < Action
+      
+      def self.run(params)
+        manager = Firebase::Manager.new
+        #Login
+        api = manager.login(params[:username])
+
+        #Select project
+        project = manager.select_project(params[:project_number])
+
+        # Client input
+        type = params[:type].to_sym
+
+        bundle_id = params[:bundle_id]
+        name = params[:name]
+        appstore_id = params[:appstore_id]
```

Comment:
I only realized know that we can pass the `appstore_id` as a param, so should remove the `store_id` variable